flocx_market/tests/unit/db/sqlalchemy/test_offer_api.py

```diff
@@ -29,12 +29,12 @@ def test_offer_create(app, db, session):
 
 
 def test_offer_create_invalid(app, db, session):
-    data = dict(test_offer_data)
-    del data['provider_id']
-    offer = OfferApi(**data)
+    del test_offer_data['provider_id']
+    offer = OfferApi(**test_offer_data)
 
     with pytest.raises(IntegrityError):
         offer.save_to_db()
+    test_offer_data['provider_id'] = '2345'
```

Comment:
I'm not clear what you're doing with these changes here. The original version was intentionally not modifying `test_offer_data` because it is used by other tests...so we create a local copy of it, and use that instead, because we can modify it as much as we want.  This change doesn't introduce any functional changes in the test, unless I'm missing something.

Reply:
Yeah that's fine, I'll remove this one